crates/re_log/src/setup.rs

```diff
@@ -49,6 +49,7 @@ pub fn setup_logging() {
             log::LevelFilter::Debug,
         )))
         .expect("Failed to install logger");
+        console_error_panic_hook::set_once();
```

Comment:
⚠️  This will conflict with the panic hook installed by `eframe` (https://docs.rs/eframe/latest/wasm32-unknown-unknown/eframe/web/struct.WebRunner.html#method.new)

Reply:
I tried removing it, reverting back to the previous egui rev, and when it panicked I got a proper call stack... It turns out that Next.js was caching the WASM from `node_modules` in its own `.next` folder, and ofc it doesn't check if it's been updated.